docs/pages/server-access/guides/openssh.mdx

```diff
@@ -340,6 +337,87 @@ authenticate the host via the certificate we generated earlier.
 
 </Details>
 
+<Details title="Using Trusted Clusters?">
```

Comment:
> It looks like the -J flag applies to the [tsh command as a whole](https://github.com/gravitational/teleport/blob/2aae5b58992dda6fcfde27ead967029a52fe098f/tool/tsh/tsh.go#L508), not just tsh proxy ssh. Should docs re: the -J flag be in a more visible location? This guide is intended for infrastructure with legacy OpenSSH servers, and it looks like the -J flag is more generally applicable.

The main intention of the proxy templates change is to `[improve] tsh proxy command interoperability with ssh client and connecting to leaf cluster proxies directly` - https://github.com/gravitational/teleport/blob/master/rfd/0062-tsh-proxy-template.md.
While the `-J` flag is usable through other commands like `tsh ssh -J {{proxy}}`, I believe the only use case we explicitly support is `tsh proxy ssh -J {{proxy}}`, which is specifically intended for use in theS SSH config's `ProxyCommand` field.

So in my opinion the OpenSSH guide is the best 

fit for the current implementation which is geared towards using the OpenSSH client to connect to OpenSSH hosts and Teleport nodes through the Teleport Auth/Proxy servers.

> It seems like there haven't been any changes to the [template used in writeSSHConfig](https://github.com/gravitational/teleport/blob/2aae5b58992dda6fcfde27ead967029a52fe098f/tool/tsh/config.go#L34-L45). Should we be instructing users to run tsh config, then manually edit the resulting config to include the ProxyCommand with the -J flag as shown below? Otherwise, I'm not sure this guide is the best place to put the -J flag instructions.

The current template is still the best default template for users to use. To use Proxy templates, they'd need to edit the config and proxy templates themselves to fit their naming conventions etc. This guide is intended for users who want to customize their ssh config beyond the default config we provide with `tsh config`.
 
To me it makes sense to provide these "next step" instructions right here. 

I'll try adding some more context in the guide, but if you have an idea for a more fitting spot in the docs we can try that.

Reply:
Thanks for clarifying! I think the best way to go would be to edit the text into a more explicit set of steps. We should tell the reader to edit the config generated by `tsh config` to use the `{{proxy}}` template, then add a `proxy_templates` field to the `tsh` config file. Right now, these sound more like general possibilities, and it can be clearer how they fit into the workflow we describe in this guide.